GameData/KerbalismConfig/Support/ROCapsules.cfg

```
@@ -601,26 +1002,47 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 8.95
-		%surface = 35
+		%volume = 9	//m^3
+		%surface = 4.3267	//m^2
```

```suggestion
		%surface = 35
```
Again, impossible for that internal volume.